<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbgraphClass</class>
 <widget class="QMainWindow" name="dbgraphClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>1055</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dbgraph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="dissassembly_tab_widget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="mnemonic_tab">
       <attribute name="title">
        <string>mnemonic</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="graph_tab">
       <attribute name="title">
        <string>graph</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dbgraph.qrc"/>
 </resources>
 <connections/>
</ui>
